lib/replies.js

```diff
@@ -70,22 +66,17 @@ module.exports.noPhotoSent = function noPhotoSent(req, res) {
   return sendResponse('invalidPhoto', req, res);
 };
 
-module.exports.campaignClosed = function campaignClosed(req, res) {
-  return sendResponse('campaignClosed', req, res);
-};
-
-module.exports.memberSupport = function memberSupport(req, res) {
-  return sendResponse('memberSupport', req, res);
-};
```

Comment:
These are more pre-TGM relics, a `POST /campaignActivity` request does not check campaign status nor for support requested commands.

Reply:
Nice pruning! ✂️